packages/terra-markdown/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "terra-markdown",
   "main": "lib/Markdown.js",
-  "private": true,
```

Comment:
Removing the private attribute. https://docs.npmjs.com/files/package.json#private